Utilities/StorageFactory/plugins/StorageAddLatencyProxy.cc

```diff
@@ -0,0 +1,130 @@
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+#include "FWCore/Utilities/interface/transform.h"
+#include "Utilities/StorageFactory/interface/Storage.h"
+#include "Utilities/StorageFactory/interface/StorageProxyMaker.h"
+
+#include <chrono>
+#include <regex>
+#include <thread>
+
+namespace edm::storage {
+  class StorageAddLatencyProxy : public Storage {
```

Comment:
Suggestions for better name for this component?

Reply:
I'm OK with this name. What is it you are uncomfortable with for this name?

Reply:
Just a feeling there could exist a better phrasing